### PR TITLE
file path-ignore in main.yml

### PR DESCRIPTION
**Pull request by shivam-sharma7**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -2,9 +2,19 @@ name: Node.js GitHub Action
...
+      - '.gitignore'
+      - '*.md'
+      - '*.txt'
+
+      
   pull_request:
+    branches: [main]
+    path-ignore:
+      - '.gitignore'
+      - '*.md'
```
Comment (MonalikaPatnaik) [inline .github/workflows/main.yml:16]: Why to ignore .md files?

Comment (shivam-sharma7) [reply .github/workflows/main.yml:16]: why we will keep it? node ci is for our front-end okay so does't make it sense to run ci whenever someone change `.md` files. simple :)

**Comment by shivam-sharma7** [comment]
@MonalikaPatnaik what do you thingh?

**Comment by shivam-sharma7** [comment]
> @MonalikaPatnaik what is your thought?

**Pull request merged**